server/flyway/sql/V55__create_permission_audit_and_code_tables.sql

```diff
@@ -0,0 +1,90 @@
+-- Create a permission audit table and a code table https://github.com/bcgov/nr-forests-access-management/issues/1536
+-- Create the fam_privilege_change_type table
+CREATE TABLE app_fam.fam_privilege_change_type (
+    privilege_change_type_code VARCHAR(10) PRIMARY KEY,
+    description VARCHAR(100) NOT NULL,
+    effective_date TIMESTAMP(6) WITHOUT TIME ZONE DEFAULT NOW() NOT NULL,
+    expiry_date TIMESTAMP(6) WITHOUT TIME ZONE,
+    update_date TIMESTAMP(6) WITHOUT TIME ZONE
+);
+
+-- Permission for fam_privilege_change_type
+GRANT
+SELECT
+    ON app_fam.fam_privilege_change_type TO ${admin_management_api_db_user}, ${api_db_username};
+
+-- Insert the values into the fam_privilege_change_type table
+INSERT INTO
+    app_fam.fam_privilege_change_type (
+        privilege_change_type_code,
+        description,
+        effective_date,
+        expiry_date,
+        update_date
+    )
+VALUES
+    (
+        'GRANT',
+        'Grant',
+        '2024-09-03',
+        '9999-01-01',
+        '2024-09-03'
```

Comment:
Probably can leave "expiry_date" and "update_date" empty?

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1577/commits/1659f872d387b2869d40383d131006a7a67df626